### ISSUE TITLE
Option to change messages

### ISSUE THREAD

**Issue by RandomAltThingy** [Hydreon/Katana#33]
Like in spigot and my fork of redstonelamp, [Spigat](http://github.com/randomaltthingy/Spigat), there should be an option to change the messages. I haven't implemented the code yet, but look in spigat.yml in the resources folder. Implement that in Katana.

**Comment by tntloverlol** [comment]
For Katana, the design philosophy as stated in the README prioritizes having things in plain english so it's easier to understand rather than going through abstraction layers. If you'd like to change what gets sent to players, do that through a plugin.
